lib/addons/topics-api.ts

```diff
@@ -0,0 +1,35 @@
+import OptableSDK from "../sdk";
+
+declare module "../sdk" {
+    export interface OptableSDK {
+        tryTopicsAPI: () => void;
+        getTopics: () => Promise<void>;
```

Comment:
You don't seem to define `getTopics`.
I actually prefer getTopics name rather than tryTopcisAPI as this is unclear what this does